README.md

```diff
@@ -78,6 +78,7 @@ Supported helpers for maps:
 - FromEntries
 - Assign (merge of maps)
 - MapValues
+- KeyBy
```

Comment:
Could you move it into the "Slice" section, please?

Also from map.go to slice.go 🙏